examples/hierarchical_pipeline.ipynb

```diff
@@ -0,0 +1,1539 @@
+{
```

Comment:
<p>There are some issues with rendering of <code>\begin{equation} &nbsp;y_{A,t} = y_{AA,t} + y_{AB,t} &nbsp;\ &nbsp;y_{t} = y_{A,t} + y_{B,t} &nbsp;\end{equation}</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1083/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Fixed